### ISSUE TITLE
[New Model]: VisionEncoderDecoderModel

### ISSUE THREAD

**Issue by papayalove** [vllm-project/vllm#8153]
### 🚀 The feature, motivation and pitch

Does vllm have transformers VisionEncoderDecoderModel? We need to do image content recognition

### Before submitting a new issue...

- [X] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://docs.vllm.ai/en/latest/), which can answer lots of frequently asked questions.

**Comment by DarkLight1337** [comment]
No. Encoder-decoder architecture is currently limited to language-only models. We have plans to support it soon since Llama 3.1 uses it.

**Comment by DarkLight1337** [comment]
Sorry, I seem to have misunderstood this model previously. It looks like this model is designed for generic multimodal encoder with generic language decoder, so it should be possible to implement in vLLM now. However, both encoder and decoder components need to be implemented in vLLM first, so it's not nearly as flexible as in `transformers`.

**Comment by DarkLight1337** [comment]
I don't think there is enough interest in this to be worth doing.
